tests/php/test_class-wc-connect-shipping-method.php

```diff
@@ -103,31 +103,6 @@ public function test_is_valid_package_destination( $package, $expected ) {
 	/**
 	 * @covers WC_Connect_Shipping_Method::calculate_shipping
 	 */
-	public function test_invalid_destination_calculate_shipping() {
```

Comment:
The test just mocks everything and asserts that the log is called, so without log, this test doesn't actually do anything.